controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -221,10 +221,11 @@ http {
         {{ $path := buildLocation $location }}
         {{ $authPath := buildAuthLocation $location }}
 
-        {{ if not (empty $location.CertificateAuth.CertFileName) }}
+        {{ if not (empty $location.CertificateAuth.CAFileName) }}
         # PEM sha: {{ $location.CertificateAuth.PemSHA }}
         ssl_client_certificate              {{ $location.CertificateAuth.CAFileName }};
         ssl_verify_client on;
+        ssl_verify_depth 10;
```

Comment:
For my own education, what is the average depth in the wild? do most people just have 1 CA that signs the 1 cert